.github/workflows/actions.yml

```diff
@@ -304,13 +304,182 @@ jobs:
+      - name: download fftw
+        shell: bash
+        env:
+          FFTW_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/fftw
+        run: |
+          mkdir -p $FFTW_PATH && cd $FFTW_PATH
+          curl -L ${{ matrix.fftw-url }} -o fftw.zip
+          7z x fftw.zip -y
+          # add env var for subsequent steps
+          echo 'FFTW_PATH<<EOF' >> $GITHUB_ENV
+          echo $FFTW_PATH >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: create fftw msvc library
+        if: env.VCVARS_SCRIPT
+        shell: cmd
+        env:
+          FFTW_ARCH: ${{ matrix.fftw-arch }}
+        run: |
+          call "%VCVARS_SCRIPT_PATH%"
+          cd "%FFTW_PATH%"
+          lib.exe /machine:%FFTW_ARCH% /def:libfftw3f-3.def
+      - name: move fftw
+        shell: bash
+        run: |
+          mv $FFTW_PATH "C:/Program Files/fftw"
+          echo "Done installing fftw"
+      - name: install asio sdk
+        shell: bash
+        env:
+          ASIO_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/asio_sdk
+        run: |
+          mkdir -p $ASIO_PATH && cd $ASIO_PATH
+          curl -L $ASIO_URL -o asio.zip
+          7z x asio.zip -y
+          mv $ASIO_FOLDER_NAME $GITHUB_WORKSPACE/external_libraries/portaudio/asiosdk
+          echo "Done installing asio sdk"
+      - name: configure
+        shell: bash
+        env:
+          CMAKE_GENERATOR: ${{ matrix.cmake-generator }}
+          CMAKE_ARCH: ${{ matrix.cmake-arch }}
+          USE_WEBENGINE: ${{ matrix.use-qtwebengine }}
+          ABLETON_LINK: ${{ matrix.ableton-link }}
+        run: |
+          export PATH=$Qt5_DIR/bin:$PATH
+
+          mkdir $BUILD_PATH && cd $BUILD_PATH
+
+          cmake -G "$CMAKE_GENERATOR" -A "$CMAKE_ARCH" -D CMAKE_PREFIX_PATH="$Qt5_DIR" -D SUPERNOVA=ON -D SC_USE_QTWEBENGINE=$USE_WEBENGINE -D SC_ABLETON_LINK=$ABLETON_LINK -D CMAKE_BUILD_TYPE=Release .. # build type is specified here for MinGW build
+
+      - name: build
+        shell: bash
+        run: |
+          cmake --build $BUILD_PATH --config Release --target install
+      - name: create archive
+        if: matrix.artifact-suffix
+        shell: bash
+        run: cd $INSTALL_PATH && 7z a $ARTIFACT_FILE.zip -tzip SuperCollider # this assumes that we end up with the build in the folder SuperCollider
+      - name: upload artifacts
+        uses: actions/upload-artifact@v2
+        if: matrix.artifact-suffix
+        with:
+          name: ${{ env.ARTIFACT_FILE }}
+          path: ${{ env.INSTALL_PATH }}/${{ env.ARTIFACT_FILE }}.zip
+          retention-days: 7 # quickly remove test artifacts
+      - name: create installer
+        if: env.CREATE_INSTALLER == 'true'
+        shell: bash
+        run: |
+          export PATH="C:\Program Files (x86)\NSIS":$PATH
+          cmake --build $BUILD_PATH --config Release --target installer
+      - name: upload installer
+        uses: actions/upload-artifact@v2
+        if: env.CREATE_INSTALLER == 'true'
```

Comment:
i think you can also replace this with `if: matrix.create-installer == 'true'`, right?